lib/go-atscfg/hostingdotconfig.go

```diff
@@ -27,15 +27,51 @@ import (
 	"github.com/apache/trafficcontrol/lib/go-tc"
 )
 
+// HostingConfigFileName is the file name of a hosting.config file.
+//
+// TODO: Remove? This is unused, even internally in this package.
 const HostingConfigFileName = `hosting.config`
+
+// HostingConfigParamConfigFile is the ConfigFile value of Parameters which can
+// affect the contents of hosting.config files.
 const HostingConfigParamConfigFile = `storage.config`
+
+// ContentTypeHostingDotConfig is the MIME type of the contents of a
+// hosting.config file.
 const ContentTypeHostingDotConfig = ContentTypeTextASCII
+
+// LineCommentHostingDotConfig is the string used to indicate the beginning of a
+// line comment in the grammar of a hosting.config file.
 const LineCommentHostingDotConfig = LineCommentHash
 
+// ParamDrivePrefix is the Name of the Parameter that determines the path prefix
+// for disk storage caching block devices.
 const ParamDrivePrefix = "Drive_Prefix"
+
+// ParamRAMDrivePrefix is the Name of the Parameter that determines the path
+// prefix for RAM caching block devices.
 const ParamRAMDrivePrefix = "RAM_Drive_Prefix"
 
+// ServerHostingDotConfigMidIncludeInactive controls whether or not to include
+// configurations for inactive Delivery Services in hosting.config for mid-tier
+// cache servers.
+//
+// TODO: remove this? It's only used in one place, where it's used as:
```

Comment:
I don't think we should remove these. They exist to convey intent, and also to make it easy and obvious how to change whether hosting.config includes inactive DSes, if someone wants or needs to fork ATC to do that: change this const.

As public symbols, they could also be used by code to determine whether inactive DSes were included. For example, ATC might want to change that behavior in the future, and someone's script might want to take a different action if that happens.